src/@apollo/client/react/hooks/ApolloClient__React_UseQuery.re

```diff
@@ -112,6 +113,7 @@ let useQuery0:
       ~pollInterval: int=?,
       ~skip: bool=?,
       ~ssr: bool=?,
+      ~variables: jsVariables=?,
```

Comment:
@jfrolich For some reason I read "no required vars" as "no vars" and `makeDefaultVariables` always returned `unit` 😅. Since that's not the case it seems to me the `0` functions should also take optional vars and `0` is maybe a bit weird naming now and we should just go with the full `useQueryNoRequiredVars`, etc. After having used the extension API for any length of time at all, I don't think anyone is going to be using this anyway.

Reply:
The reason for `useQuery0` not taking in any variables is that you don't pass in variables, only working if there are no required variables. If you **want** to pass in variables you can use the regular `useQuery`.

Reply:
The reason I used `useQuery0` is because I found `useQueryNoRequiredVars` a bit long :) But maybe there is a better name that is also short.

Reply:
I think my headspace was still in the land of "no vars". Now that my head is straight. I think we keep the `0` naming. It holds up when there are no variables arguments.